### PR TITLE
Stepped Functions & Unset Value to Preset Env/CLI

### PR DESCRIPTION
**Pull request by Antonio-Laguna**
This takes care of pushing the new package to preset env and the cli and also keeping up to date preset env's site.

If this is good, preset env will be released and then CLI and the site will be both updated and CLI released.

### CODE REVIEW

**Comment by romainmenke** [comment]
Can we also add `postcss-unset-value` to preset-env?

_Hadn't done that yet as I was hoping to have some color related work done more quickly._

**Comment by Antonio-Laguna** [comment]
@romainmenke yep, doing that!

File: cli/csstools-cli/src/plugins/postcss-unset-value.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import plugin from '@csstools/postcss-unset-value';
+import { cli, helpTextLogger } from '@csstools/base-cli';
+
+export default function postcssUnsetValue() {
+	cli(
+		plugin,
+		['preserve', 'onInvalid'],
```
Comment (romainmenke) [inline cli/csstools-cli/src/plugins/postcss-unset-value.ts:7]:
```suggestion
		['preserve'],
```

**Review by romainmenke — COMMENTED** [review]
One small thing, rest looks perfect 🎉

**Review by romainmenke — APPROVED** [review]
LGTM!

Thank you for both of these 🎉

**Pull request merged**